src/main/java/cn/edu/tsinghua/iotdb/benchmark/workload/SyntheticWorkload.java

```diff
@@ -48,6 +49,24 @@ public SyntheticWorkload(int clientId) {
     }
   }
 
+  private static String[][] initWorkloadValues () {
+    String[][] workloadValues = null;
+    if(!config.OPERATION_PROPORTION.split(":")[0].equals("0")) {
+      workloadValues = new String[config.SENSOR_NUMBER][config.BATCH_EXECUTE_COUNT];
```

Comment:
should here use BATCH_SIZE ? What is BATCH_EXECUTE_COUNT mean?

Reply:
fixed, use WORKLOAD_BUFFER_SIZE